src/Simplex/Chat/Controller.hs

```diff
@@ -201,7 +226,20 @@ instance ToJSON HelpSection where
   toEncoding = J.genericToEncoding . enumJSON $ dropPrefix "HS"
 
 data ChatCommand
-  = ShowActiveUser
+  = CreateRemoteHost -- ^ Configure a new remote host
+  | ListRemoteHosts
+  | StartRemoteHost RemoteHostId -- ^ Start and announce a remote host
+  | StopRemoteHost RemoteHostId -- ^ Shut down a running session
+  | DisposeRemoteHost RemoteHostId -- ^ Unregister remote host and remove its data
+  | StoreRemote FilePath RemoteHostId -- ^ Store a file at host, preparing for `SendFile`
+  | FetchRemote FilePath RemoteHostId -- ^ Fetch a file from host after `ReceiveFile`
+  | RegisterRemoteController Text -- ^ Register OOB data for satellite discovery and handshake
+  | StartRemoteController -- ^ Start listening for satellite announcements
+  | ConfirmRemoteController -- ^ Confirm discovered data and store confirmation
+  | RejectRemoteController -- ^ Reject discovered data (and blacklist?)
+  | StopRemoteController -- ^ Stop listening for announcements or terminate an active session
+  | DisposeRemoteController Text -- ^ Remove all local data associated with a satellite session
```

Comment:
Above is not clear. There can be one remote controller (desktop) active at a time, but there can be multiple remote controllers registered with the host, so IDs seem missing in at least some commands. It's only when you add a new one you will get ID in response. Also there should be command for listing them and de-registering them. Above seems to be mixing sessions and known remote controllers.

Reply:
The 3 commands without arguments are working on a current session, initiated by getting OOB from UI.

The de-registering command is `Dispose`. It works on a persisted state, thus needs an identifier.